content/posts/Harsh_Phase_1_Report.md

```diff
@@ -0,0 +1,80 @@
+| **9 – 15 Jun (Week 1)**     | ViewComponent setup & Header migration         | This week, I initiated the project by migrating the primary Header and Notification partials into dedicated ViewComponents. A key part of this effort was integrating Right-to-Left (RTL) language support into the new header structure. To facilitate a smoother review process, I decomposed the large header pull request into smaller, more focused PRs. |
+| **16 – 22 Jun (Week 2)**    | Core Card migrations & search discovery        | The focus shifted to core content elements, with the Project Card and User Card being successfully migrated to ViewComponents. I also completed migrations for the Contribute Card and site-wide Pagination. During this process, I identified a useful, undocumented feature for filtering projects by tags directly through URL patterns.                   |
+| **23 – 29 Jun (Week 3)**    | PR decomposition & Search Bar componentization | To better manage the ongoing header work, I split the task into two distinct pull requests: one for the Search Bar migration and another for the Header UI revamp. The Search Bar was successfully extracted into its own reusable ViewComponent, and the main Header UI was updated to align with the new Figma designs.                                     |
+| **30 Jun – 6 Jul (Week 4)** | Feedback resolution & multiple PR merges       | This was a week of consolidation. I dedicated time to addressing feedback and resolving all open comments on the Contribute Card pull request, which led to the successful merge of the Contribute, Project, and User Card migration PRs.                                                                                                                     |
+| **7 – 13 Jul (Week 5)**     | UI revamps merged & Home Page drafting         | Significant progress was made on the UI front. The Project Card and Search Bar UI revamps were finalized and merged into the main branch. I opened a new pull request for the User Card revamp, which is close to being resolved, and also began drafting a PR for the Home Page revamp, starting with the hero section.                                      |
+| **14 – 15 Jul (Week 6)**    | Landing Page PR & User Card wrap‑up            | In the final week of Phase 1, I created the official pull request for the Landing Page (Home) revamp, ensuring it included comprehensive RTL support from the start. Additionally, I resolved all remaining comments on the User Card revamp PR, preparing it for its final merge.                                                                            |
+
+---
+
+## UI Revamps
+
```

Comment:
Could you briefly describe why it was needed in the first place -> just keep it short!